lib/assets/javascripts/utils/tableHelper.js

```diff
@@ -42,16 +43,10 @@ export const filteriseTable = (options) => {
       debounced(e.currentTarget);
     });
 
-    $(options.selector).siblings('#clear_filter').click((e) => {
+    $('.clear_filter').click((e) => {
       e.preventDefault();
-      clear(e.currentTarget);
+      clear(e.target);
```

Comment:
Just curious, but why switch to `target` here but not elsewhere (like debounced above)? I just looked up the Mozilla docs again and see that `currentTarget` returns the EventTarget instead of the DOM element so `target` does seem more appropriate.
https://developer.mozilla.org/en-US/docs/Web/API/Event/Comparison_of_Event_Targets

Reply:
In this case makes no different one or the other since both return the same element, i.e. the 'a' element in which the handler is attached. The use of currentTarget is more associated with event delegators, really useful for listening to events even on future elements added underneath. The keyup handler where the debounced function is invoked similarly could be changed to target and would work. I would be more worried about declaring an anonymous function anytime keyup is triggered but at this point, not worth to change it.